service-front/module/Application/view/application/authenticated/lpa/primary-attorney/partials/attorney.twig

```diff
@@ -4,10 +4,10 @@
         <p class="person-address flush--ends">{{ attorney['address'] }}</p>
         <ul class="person-actions">
             <li class="action-item">
-                <a href="{{ editUrl }}" role="button" class="action-edit js-form-popup" aria-label="View/edit Attorney, {{ attorney['name'] }}">View/edit details</a>
+                <a href="{{ confirmDeleteUrl }}" class="action-delete js-form-popup">Delete</a>

```

Comment:
This element needs to be moved after the edit button. The focus still moves incorrectly between the buttons (from right to left).

Reply:
It might even be possible to add a cypress test for this, which I'd recommend, to prove the fix.

Reply:
Actually, I can see I'm going to have some fun merging with this branch :)

Reply:
@Raffers This issue is also present on the attorneys page. I'd recommend searching for every page with a View/edit link and modifying all of them. (I know this is outside scope of the issue, but only because there isn't a single partial which does all of the view/edit lists, which is what I was hoping for.)

Reply:
Also note that the float needs to stay float:left, it's actually the element order which is causing them the order on the page to be incorrect.

Reply:
And I also noticed that the HTML appears to be wrong on this page.

To summarise, this is what I see on replacement attorneys:

```
        <ul class="person-actions">
            <li class="action-item">
                <a href="/lpa/52614158710/replacement-attorney/confirm-delete/0" class="action-delete js-form-popup">Delete</a>
            </li>
                <a href="/lpa/52614158710/replacement-attorney/edit/0" role="button" class="action-edit js-form-popup" aria-label="View/edit Replacement Attorney Mr Dead Pool">View/edit details</a>
            <li class="action-item">
            </li>
        </ul>```

with this CSS applied to `.action-item`:

```
@media (min-width: 641px)
.person-actions .action-item {
    margin-left: 8px;
    float: right;
}
```

when what we need is this:

```
        <ul class="person-actions">
            <li class="action-item">
                <a href="/lpa/52614158710/replacement-attorney/edit/0" role="button" class="action-edit js-form-popup" aria-label="View/edit Replacement Attorney Mr Dead Pool">View/edit details</a>
            </li>
            <li class="action-item">
                <a href="/lpa/52614158710/replacement-attorney/confirm-delete/0" class="action-delete js-form-popup">Delete</a>
            </li>
        </ul>```

with this CSS:

```
@media (min-width: 641px)
.person-actions .action-item {
    margin-left: 8px;
    float: left;
}
```

Reply:
Just realised this is wrong: Delete is to the left of View/edit in the live site (seems wrong to me, but there you go). So ignore the above and I'll approve this PR.